src/api/app/policies/decision_policy.rb

```diff
@@ -4,4 +4,14 @@ def create?
 
     user.is_moderator? || user.is_admin? || user.is_staff?
   end
+
+  def appeal?
```

Comment:
So this needs to be done here since to create an appeal, we first need to check if a decision was in favor of the report. Then, we check if the first report can be updated by the current user. Why only the first report? What if this report wasn't from the current user?

Reply:
So from my understanding, a decision can have many reports. Those reports should always belong to the same reportable. Thats why I only check permission based on the first report. I check if the user has permission to update the reportable, because that would mean IMO that the user should also be able to appeal the decision. I think we don't have yet a validation in place to make sure the reports of a decision always point to the same reportable, but technically there is not really a way to do that (other then having access to the console and doing it by hand). Until we have the validation, I would also be fine to go through all reports and do the permission check. What do you think? This is the card https://trello.com/c/xrjOZGa7/45-ensure-all-reports-of-a-decision-point-to-the-same-reportable-m

Reply:
This is handled here now https://github.com/openSUSE/open-build-service/pull/15342